test/compilable/extra-files/objdump-postscript.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+expect_file=${EXTRA_FILES}/${TEST_NAME}.out
+obj_file=${OUTPUT_BASE}_0.o
+
+echo Creating objdump
+objdump --disassemble --disassembler-options=intel-mnemonic "${obj_file}" > "${obj_file}.dump"
```

Comment:
These options are not recognized on macOS.

Reply:
The test isn't run on macOS (it's Linux only.)

Reply:
Ok.